doc/user/content/ops/network-security/privatelink.md

```diff
@@ -213,6 +213,17 @@ CREATE SOURCE mz_source
 {{< /tab >}}
 {{< /tabs >}}
 
+## Terraform Modules
```

Comment:
It'd maybe be sufficient to include a link to the Terraform modules as a note under the MSK tab? If we start seeing traction and decide to be more intentional about supporting these, we can make them more prominent elsewhere in the documentation.

Reply:
The issue is we will never make ourselves liable if someone runs terraform and then claims we mucked up their aws infra. We need to be very thoughtful about the warning.

Reply:
Exactly. That's why I'm leaning towards reducing the diff to an annotation that just points to the module repo (where we can have all the required 🚨🚨) rather than an explicit section.

Reply:
I don't have a strong opinion about this. Open to suggestions on where to put that note, and do we want to to be an actual note or just a plain paragraph, eg:
<img width="887" alt="image" src="https://user-images.githubusercontent.com/21223421/215986226-ead703e2-045c-4f8f-baa6-e6aa502236a1.png">